client.go

```diff
@@ -241,6 +262,24 @@ func (c *Client) postJSON(ctx context.Context, config *apiConfig, apiReq interfa
 	return json.NewDecoder(httpResp.Body).Decode(resp)
 }
 
+func (c *Client) setExperienceId(ids ...string) {
+	c.experienceId = ids
+}
+
+func (c *Client) getExperienceId() []string {
+	return c.experienceId
+}
+
+func (c *Client) clearExperienceId() {
+	c.experienceId = nil
+}
+
+func (c *Client) setExperienceIdHeader(req *http.Request) {
+	// if len(c.experienceId) > 0 {
+	req.Header.Set(EXPERIENCE_ID_HEADER_NAME, strings.Join(c.experienceId, ","))
```

Comment:
Should the header not be set at all if no experience ID is supplied (i.e. uncomment line above)?

Reply:
thanks for the catch, i need to revert that